imperial.js

```diff
@@ -29,50 +30,17 @@ class Imperial {
 
     if (lastMoveSkippedInvestorSlot) {
       return [
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.AH, player: "Claudia", cost: 4 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.AH, player: "Claudia", cost: 6 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.IT, player: "Claudia", cost: 2 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.IT, player: "Claudia", cost: 4 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.IT, player: "Claudia", cost: 6 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.FR, player: "Claudia", cost: 4 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.FR, player: "Claudia", cost: 6 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.GB, player: "Claudia", cost: 4 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.GB, player: "Claudia", cost: 6 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.GE, player: "Claudia", cost: 2 },
-        },
-        {
-          type: "bondPurchase",
-          payload: { nation: Nation.RU, player: "Claudia", cost: 4 },
-        },
+        bondPurchase(Nation.AH, "Claudia", 4),
```

Comment:
I love the shorter syntax! I wonder if we might be losing something by not having the arguments have keywords? It might get confusing as we add more actions. What do you think?

Reply:
I'm definitely up to use keyword-style arguments! I'll make that change.